docs/modules/deployment/pages/upgrade/basic.adoc

```diff
@@ -0,0 +1,134 @@
+
+[[upgrade-basic]]
+= Basic Upgrade
+
+This procedure describes how to do a basic {page-component-title} upgrade, including upgrading the PostgreSQL database to a new version.
+You may need to complete additional steps to upgrade in a more complex setup (running more than one OpenNMS instance, database migration requirements, age of current version, and so on).
+
+IMPORTANT: Make sure you complete the tasks in the xref:deployment:upgrade/introduction.adoc#byb_upgrade[before you begin] section before upgrade.
+
+== PostgreSQL upgrade and configuration
+This example illustrates an upgrade from PostgreSQL 9.4 to 12.
+The process should be similar for different versions.
+
+. Become root:
+`sudo su -`
++
+Expected outcome: `whoami` returns `root`.
+
+. Install the latest version of the PostgreSQL repository package:
+`yum -y install <repository package location>`
```

Comment:
Maybe this should be linked to https://www.postgresql.org/download/linux/redhat/ instead, to be more general?  Not sure how that should be written, though.